integration-tests/src/user/runtime_user.rs

```diff
@@ -169,15 +169,14 @@ impl RuntimeUser {
             block_hash: Default::default(),
         }];
         for hash in &receipt_ids {
-            transactions
-                .extend(self.get_recursive_transaction_results(&hash.clone().into()).into_iter());
+            transactions.extend(self.get_recursive_transaction_results(&(*hash)).into_iter());
```

Comment:
`&*hash` should just be `hash`.

Reply:
@mina86 Yes, it's not needed, it's strange that clippy doesn't warn about it.

Reply:
I created an issue at https://github.com/rust-lang/rust-clippy/issues/8149